src/backend/src/prisma/schema.prisma

```diff
@@ -923,3 +960,45 @@ model Milestone {
   organizationId String
   organization   Organization @relation(fields: [organizationId], references: [organizationId])
 }
+
+model Graph {
+  organizationId        String           @unique
+  startDate             DateTime
+  endDate               DateTime
+  title                 String
+  linkId                String           @id @default(uuid())
+  graphType             Graph_Type
+  cars                  Car[]
+  teams                 Team[]
+  projects              Project[]
+  userCreated           User             @relation(fields: [userCreatedId], references: [userId], name: "graphCreator")
+  userCreatedId         String
+  userDeleted           User?            @relation(fields: [userDeletedId], references: [userId], name: "graphDeleter")
+  userDeletedId         String?
+  dateDeleted           DateTime?
+  graphDataId           String           @unique
+  data                  GraphData        @relation(fields: [graphDataId], references: [id])
+  groupBy               Data_Type
+  GraphCollection       GraphCollection? @relation(fields: [graphCollectionLinkId], references: [linkId])
+  graphCollectionLinkId String?
+}
+
+model GraphData {
+  id     String    @unique
+  type   Data_Type
+  values Values
+  Graph  Graph[]
```

Comment:
This indicates a one to many relation but the graph model shows a unique id for graphdata